features/admin/admin_journey.feature

```diff
@@ -22,20 +22,6 @@ Scenario: As an admin user who has logged in to Digital Marketplace, I wish to s
   And I click the search button for 'service_id'
   Then I am presented with the summary page for that service
 
-Scenario: As an admin user who has logged in to Digital Marketplace, I wish to search for services by DUNS number
-  Given I have logged in to Digital Marketplace as a 'Administrator' user
-  When I enter '930123456789' in the 'supplier_duns_number' field
-  And I click the search button for 'supplier_duns_number'
-  And I am taken to the 'Suppliers' page
-  And I click the 'Services' link
-  Then  I am presented with the 'Services' page for the supplier 'DM Functional Test Supplier'
-  And I can see all listings ordered by lot name followed by listing name
```

Comment:
If we're removing these steps then we should also delete the unused step definitions. (At least, I assume things like  "I can see all listings ordered by lot name followed by listing name" and "I am presented with the service page for that specific listing" are specific to just these steps.)